### PR TITLE
feat: propose adding Zach Newman to list of maintainers

### PR DESCRIPTION
**Pull request by trishankatdatadog**
Please fill in the fields below to submit a pull request.  The more information that is provided, the better.

Fixes #<Issue>
Release Notes: 

**Types of changes**:

- [  ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [  ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

**Description of the changes being introduced by the pull request**:

**Please verify and check that the pull request fulfills the following requirements**:

- [N/A] Tests have been added for the bug fix or new feature
- [N/A] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -3,4 +3,5 @@ Trishank Karthik Kuppusamy <trishank.kuppusamy@datadoghq.com> (github: trishanka
 Joshua Lock <jlock@vmware.com> (github: joshuagl)
 Ethan Lownman <ethan.lowman@datadoghq.com> (github: ethan-lowman-dd)
 Marina Moore <mm9693@nyu.edu> (github: mnm678)
+Zach Newman <zjn@chainguard.dev> (github: znewman01)
```
Comment (asraa) [inline MAINTAINERS:6]: Zack or Zach? @znewman01

**Review by asraa — APPROVED** [review]

File: MAINTAINERS (hunk 1 above)
Comment (znewman01) [reply MAINTAINERS:6]: Zack :) but will answer to either!

**Review by znewman01 — DISMISSED** [review]

**Review by ethan-lowman-dd — DISMISSED** [review]

File: MAINTAINERS (hunk 1 above)
Comment (trishankatdatadog) [reply MAINTAINERS]: My bad

**Review by znewman01 — APPROVED** [review]

**Review by mnm678 — APPROVED** [review]

**Review by joshuagl — APPROVED** [review]

**Comment by trishankatdatadog** [comment]
@ethan-lowman-dd or @hosseinsia: are either of you happy to ✅ ? Then we can merge and 🚀

**Review by ethan-lowman-dd — APPROVED** [review]

**Comment by znewman01** [comment]
Anything blocking merging this? There's a few other PRs in the queue and I'd like to take a look at approving them.

**Pull request merged**